### PR TITLE
first commit for main.go API

### PR DESCRIPTION
**Pull request by snkzt**
Created main.go for dealing with API access

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	}
+
+	defer db.Close()
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/lights", getLights).Methods("GET")
+	router.HandleFunc("/lights/create", createLights).Methods("POST")
+	router.HandleFunc("/lights/update", updateLights).Methods("POST")
+	router.HandleFunc("/lights/delete", deleteLights).Methods("POST")
```
Comment (VeysonD) [inline main.go:36]: Shouldn't this be DELETE method?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+		panic(err.Error())
+	}
+
+	defer db.Close()
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/lights", getLights).Methods("GET")
+	router.HandleFunc("/lights/create", createLights).Methods("POST")
+	router.HandleFunc("/lights/update", updateLights).Methods("POST")
```
Comment (VeysonD) [inline main.go:35]: You can use `PUT` here instead

File: main.go (hunk 1 above)
Comment (snkzt) [reply main.go:36]: Okay! Changed to Delete method!
Would be updated with the next commit!

File: main.go (hunk 2 above)
Comment (snkzt) [reply main.go:35]: Okay! Changed to Put method!
Would be updated with the next commit!

**Pull request merged**